style.css

```diff
@@ -0,0 +1,585 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Lato, Arial, sans-serif;
+  list-style: none;
+}
+
+hr {
+  border: 1px solid #a04038;
+  background-color: #a04038;
+  width: 30px;
+  height: 2px;
+  margin-bottom: 20px;
+  margin-top: 15px;
+}
```

Comment:
- [ ] Kindly make sure to stick to the given color palette when ever implementing them in your design as I noticed the horizontal line color is different from the one shown in the Behance design
- Behance design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/336721f6-b591-4f56-9042-fa40e85452fe)
- Your design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/5b07bd64-22a2-41ef-a623-0ac1edb727e2)

- Basic color swatch: `#272A31`, `#EC5242`, `#D3D3D3`